app/services/scoring/player_svc.rb

```diff
@@ -0,0 +1,19 @@
+class Scoring::PlayerSvc
+
+    #calculates the points earned by player in match
+    def self.call(player, match)
+
+        #list of all of player's player_stats from match
+        player_stats = match.player_stats.select {|stat| stat.player_id == player.id}
+
+        #list of all stat_ids for player from match
+        only_stats = player_stats.map {|player_stat| player_stat.stat_id}
+
+        #list of the point value for each stat for plaeyr in match, just need to add it up
+        stat_values = only_stats.map {|id|   Stat.where("id = ?", id).points}
```

Comment:
Simplified as:
```
stat_values = Stat.where(id: only_stats).pluck(:points)
```

Reply:
so this finds each stat whose ID is in only_stats? only_stats is a list of ID's, not just one ID, unlike usual